pages/api/integrations/googlecalendar/add.ts

```diff
@@ -19,16 +18,6 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       return;
     }
 
-    // Get user
-    const user = await prisma.user.findFirst({
-      where: {
-        email: session.user.email,
-      },
-      select: {
-        id: true,
-      },
-    });
-
```

Comment:
cc @baileypumfleet you know why this was here? maybe that it was unused obscures a bug

Reply:
cc @emrysal

Reply:
@Alex / KATT 5 months ago, you're looking at history being made there - but no, that user should not be in add.ts - I think it's a copy-paste from callback.ts. Never spotted, that file has been basically unmodified for most of it's history and it never was a problem - so that's why,

https://github.com/calendso/calendso/commit/1582cfd43f21a88cd2eec9e387d8e7e971bb90a4

Given ID is now part of the user session, in addition to never needing that call in add.ts, we can also remove it from callback.ts (and get the id from the session).